src/Compilers/CSharp/Test/Symbol/Symbols/Source/RecordTests.cs

```diff
@@ -1231,25 +1228,10 @@ public void DataClassAndStruct()
                 // (3,14): error CS8803: Top-level statements must precede namespace and type declarations.
                 // data class C2(int X, int Y);
                 Diagnostic(ErrorCode.ERR_TopLevelStatementAfterNamespaceOrType, "(int X, int Y);").WithLocation(3, 14),
-                // (3,14): error CS0201: Only assignment, call, increment, decrement, await, and new object expressions can be used as a statement
```

Comment:
>// (3,14): error CS0201: Only assignment, call, increment, decrement, await, and new object expressions can be used as a statement [](http://example.com/codeflow?start=16&length=130)

Are these errors no longer reported? Otherwise please place them back into the original position in the base-line.

Reply:
Fixed.